CI/macos/build_librist.sh

```diff
@@ -88,12 +119,16 @@ build-librist-main() {
     fi
 
     NOCONTINUE=TRUE
+    PRODUCT_REPO="librist"
+    PRODUCT_PROJECT="librist"
     PRODUCT_URL="https://code.videolan.org/rist/librist/-/archive/v${PRODUCT_VERSION:-${CI_PRODUCT_VERSION}}/librist-v${PRODUCT_VERSION:-${CI_PRODUCT_VERSION}}.tar.gz"
```

Comment:
`PRODUCT_URL` and `PRODUCT_FILENAME` are not needed when using git checkout.

Reply:
TBH I don't know if it might interfere otherwise, shell scripts aren't the most robust things. 😂

If we need to revert, we can always look/copy from the earlier commit, that's what the git history is for.